Mage.Tests/src/test/java/org/mage/test/cards/single/khm/MaskwoodNexusTest.java

```diff
@@ -13,6 +13,15 @@
 import org.junit.Test;
 import org.mage.test.serverside.base.CardTestPlayerBase;
 
+/**
+ * {@link mage.cards.m.MaskwoodNexus Maskwood Nexus}
+ * {4}
+ * Artifact
+ * Creatures you control are every creature type. The same is true for creature spells you control and creature cards you own that aren't on the battlefield.
+ * {3}, {T}: Create a 2/2 blue Shapeshifter creature token with changeling.
+ * @author TheElk801
```

Comment:
author tag should be you if you wrote the test

Reply:
Ooh I thought it was the person who wrote the card. I'll up date it when I'm back